src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -1122,7 +1122,11 @@ IOperation statement
             )
         {
 #if DEBUG
-            Debug.Assert(spillingTheStack || _evalStack.All(o => o.Kind == OperationKind.FlowCaptureReference || o.Kind == OperationKind.DeclarationExpression || o.Kind == OperationKind.Discard));
+            Debug.Assert(spillingTheStack || _evalStack.All(
+                o => o.Kind == OperationKind.FlowCaptureReference 
+                || o.Kind == OperationKind.DeclarationExpression
+		|| o.Kind == OperationKind.Discard
```

Comment:
>		 [](start = 0, length = 2)

Note that you used tabs here, which means that it may have looked consistent on your screen but not on other screens with different tab stop widths configured.